packages/core/src/app/shipping/PayPalFastlaneShippingAddress.tsx

```diff
@@ -1,13 +1,39 @@
-import { Address } from '@bigcommerce/checkout-sdk';
-import React, { FC } from 'react';
+import {
+    Address,
+    Consignment,
+    Country,
+    CustomerAddress,
+    FormField,
+    ShippingInitializeOptions
+} from '@bigcommerce/checkout-sdk';
+import React, {FC, useEffect, useRef, useState} from 'react';
 
-import { PayPalFastlaneShippingAddressForm } from '@bigcommerce/checkout/paypal-fastlane-integration';
+import {
+    isPayPalCommerceFastlaneMethod,
+    isPayPalFastlaneMethod,
+    PayPalFastlaneShippingAddressForm,
+    usePayPalFastlaneAddress,
+} from '@bigcommerce/checkout/paypal-fastlane-integration';
```

Comment:
Can we move this to `paypal-fastlane-integaration` package?

Reply:
We can't move it there since we will not have access to core Shipping component used in this PR